tests/fuzz.py

```diff
@@ -249,32 +261,45 @@ def fuzz(tests_dir, fuzz_arg, num_iter):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Fuzzing KECC.')
-    parser.add_argument('-n', '--num', type=int, help='The number of tests')
+    parser.add_argument('-n', '--num', type=int, help='The number of tests', default=None)
```

Comment:
I think we do not need this change because default value of `default` argument  is `None`. (https://docs.python.org/3/library/argparse.html#default)

Reply:
Oh, I didn't notice that, I will fix it